### ISSUE TITLE
Comments Shouldn't Be Evaluated

### ISSUE THREAD

**Issue by JRJurman** [Tram-One/tram-one#92]
## Summary
It appears that the combination of hyperx and other tools causes the following line to be evaluated in odd ways.

```javascript
const Tram = require('tram-one')
const html = Tram.html()

const foo = () => html`
  <div>
    <!-- <foo/> -->
  </div>
`

console.log(foo().outerHTML)
```

Instead of returning `<div><!--<foo/>--></div>`
we get  `<div><!--undefined--></div>`

In other instances this has cause hyperx to throw an error... In either scenario we should not be evaluating commented code, if possible.

**Comment by JRJurman** [comment]
This example specifically throws an unfriendly error:

```javascript
const Tram = require('tram-one')
const html = Tram.html()

const value = "thing"

const foo = () => html`
  <div>
    <!-- <foo prop=${value}>values</foo> -->
  </div>
`

console.log(foo().outerHTML)
```

**Comment by JRJurman** [comment]
This issue is resolved in the newest version of `hyperz` and will be merged in with #100
